src/util/performWithRetries.js

```diff
@@ -0,0 +1,23 @@
+import assign from 'lodash/assign';
+import promiseRetry from 'promise-retry';
+
+export default function performWithRetries(
+  perform,
+  errorsToRetry = [],
```

Comment:
I think this should probably be a function that is passed an error and returns `true` if the error merits a retry